docs/docs/user-guide/using-cli-mcp-server.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Use CLI for Microsoft 365 MCP Server
+sidebar_position: 14
+---
+
+# Use CLI for Microsoft 365 MCP Server
+
+CLI for Microsoft 365 MCP Server allows you to natural language to manage your Microsoft 365 tenant. It is a wrapper around the CLI for Microsoft 365 commands, enabling you to use natural language which will be translated into the appropriate CLI for Microsoft 365 command or chain of commands to fulfill your request.
+
+For latest updates and guides, please refer to the [CLI for Microsoft 365 MCP Server repo](https://github.com/pnp/cli-microsoft365-mcp-server).
+
+## Installation
+
+Before you start please be sure you have the following prerequisites:
+
+- Node.js 20.x or higher
+- CLI for Microsoft 365 installed globally (npm i -g @pnp/cli-microsoft365@latest)
+- CLI for Microsoft 365 setup with your Microsoft 365 tenant - you may use [m365 setup](../cmd/setup) command
+
+After you have done the prerequisites, you should update the CLI for Microsoft 365 configuration for best experience with MCP Server. Run the following commands:
+
+```powershell
+m365 cli config set --key prompt --value false
+m365 cli config set --key output --value text
+m365 cli config set --key helpMode --value full
+```
+
+This will ensure that the MCP server will get as much information as possible from the CLI for Microsoft 365 when an error occurs or when a command is not executed properly.
+
+The MCP Server is installed as a Node.js package. You can install it globally using the following command:
+
+The MCP server will not do any authentication for you. You will need to first authenticate using CLI for Microsoft 365 using [m365 login](../cmd/login) command. Once you are authenticated, the MCP server will use the same authentication context when running any tool.
+
+## Add MCP to VS Code
+
+This MCP server may work with any MCP client you prefer. To do that refer to the instructions of the picked MCP host. Below, you may find guidance for VS Code.
+
+1. Open the Command Palette (Ctrl+Shift+P or Cmd+Shift+P on macOS) and type MCP: Add Server.
+2. Select Command (stdio) as the server type.
+3. Enter the command to run the MCP server, which is:
+
+```powershell
+npx -y @pnp/cli-microsoft365-mcp-server@latest
+```
```

Comment:
If we give this code fence a few indents to the right, it will go under the 3rd bullet instead of standing apart from the bullet list.